docs/GettingStarted/QuickstartTutorial.md

```diff
@@ -1,280 +1,370 @@
+* A cup of tea, coffee or other favourite drink 
 
-Ensure that your github fork of mig-demo-apps is correctly set as the uri value in the todolist BuildConfig, by running
-```
-oc get buildconfig.build.openshift.io/todolist --namespace mongo-persistent -o=go-template='URI value: {{.spec.source.git.uri | printf "%s\n"}}'
-```
+### GitHub project fork
 
-### Pelorus configuration
+The first step is to [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) our repository with sample application to your personal [GitHub](https://github.com) space. Use the link below that opens a "new Fork" page:
 
-After following [Installation guide](../Installation/), copy `values.yaml` file to `/var/tmp/values.yaml`, by running
-```
-cp charts/pelorus/values.yaml /var/tmp/
+: [Create a fork: https://github.com/konveyor/mig-demo-apps](https://github.com/konveyor/mig-demo-apps/fork)
+
+> **IMPORTANT**: From now on **`<your_org>`** used in this document will be your GitHub **`username`** where the **`mig-demo-apps`** was forked and should replace the **`konveyor`** part of the URL.
+
+The [Mean Time to Restore and Change Failure Rate](#mean-time-to-restore-and-change-failure-rate) part of this demo uses GitHub Issues provider, so enable the **Issues** in the forked repo under the settings page:
+
+: **https://github.com/`<your_org>`/mig-demo-apps/settings#features**
+
+### Pelorus Operator
+
+Deployment of the running Pelorus Operator instance consists of two steps:
+
+ * [Installing Pelorus Operator](#installing-pelorus-operator)
+ * [Creating Pelorus instance](#creating-pelorus-instance)
+ : This step requires a [configuration file](#prepare-config-file), so let's start with this it first.
+
+#### Prepare Config file
+
+> **NOTE**: A comprehensive list of all configuration options is available in our [Pelorus Core](./configuration/PelorusCore.md) and [Pelorus Exportrs](./configuration/PelorusExporters.md) documentation.
+
+For this demo we will need to prepare a configuration file that will be used in our deployment. For now it's enought to understand that our Pelorus Deployment will consist of:
+
+* The pelorus instance:
+    * is named **pelorus-quickstart**
+* Pelorus will deploy three exporters:
+    * **deploytime** to watch `mongo-persistent` namespace
+    * **committime** to watch `mongo-persistent` namespace
+    * **failure** to monitor our application failures logged in the [Forked GitHub project](#github-project-fork). For this step you need to adjust the configuration file to match the `<your_org>` and GitHub [Personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+
+Let's create the Pelorus Configuration file and save it under the `pelorus-quickstart.yaml` file. Please adjust the `<your_github_api_token>` and `<your_org>`:
+
+```shell
+$ cat > pelorus-quickstart.yaml <<EOF
```

Comment:
this way is faster to do it, but I would put the yaml content and tell user to create a file with that content, because it is easier to read

Reply:
ok changed